src/commands/server/deploy.ts

```diff
@@ -274,51 +274,84 @@ export default class Deploy extends Command {
       this.warn('"--domain" flag is ignored for Openshift family infrastructures. It should be done on the cluster level.')
     }
 
-    if (flags.installer) {
-      if (flags.installer === 'minishift-addon') {
-        if (flags.platform !== 'minishift') {
-          this.error(`🛑 Current platform is ${flags.platform}. Minishift-addon is only available for Minishift.`)
-        }
-      } else if (flags.installer === 'helm') {
-        if (flags.platform !== 'k8s' && flags.platform !== 'minikube' && flags.platform !== 'microk8s' && flags.platform !== 'docker-desktop') {
-          this.error(`🛑 Current platform is ${flags.platform}. Helm installer is only available on top of Kubernetes flavor platform (including Minikube, Docker Desktop).`)
-        }
+    if (flags.installer === 'helm') {
+      if (flags.platform !== 'k8s' && flags.platform !== 'minikube' && flags.platform !== 'microk8s' && flags.platform !== 'docker-desktop') {
```

Comment:
We can simplify a bit:

!isKubernetesPlatformFamily(flags.platform) && flags.platform !== 'docker-desktop'